### DISCUSSION TITLE
[灌水]大新闻！珀尔曼跑了

### DISCUSSION THREAD

**Discussion by JTLIYOO** [share121/inter-knot#631] [其他]
天啊大家看新闻了吗，珀尔曼在远景案的开庭路上逃跑了！新闻说飞艇的信号飞到外环之后消失了，治安局已经发通缉令了！
![16907abc-d6e4-44ea-8ac1-e27a43c625e1](https://github.com/user-attachments/assets/7b086aa5-8430-43e9-96bd-5b7ad42449f3)

**Comment by ZhuYuan-148** [comment]
治安局不会放过任何一个法外之徒，请市民们安心

**Comment by langxmo** [comment]
要我说，空艇全员被麻醉，唯一清醒的人力挽狂澜驾驶空艇迫降才是最震撼的

**Comment by SWYH134649** [comment]
在现场！当时在带盗洞客出空洞，那个飞艇差点就撞上那一栋大楼了
